File: prototype/ts/ResultCloud.ts
import { ExtendedModel, model, prop } from 'mobx-keystone';

import Onto from './Onto';
import NormalPoint from './NormalPoint';
import ResultCloudData from './ResultCloudData';
import { ResultCloud as IResultCloud } from './types';


@model('onto/ResultCloud')
class ResultCloud extends ExtendedModel(Onto, {
    ResultCloudData: prop<ResultCloudData>(() => new ResultCloudData({})),
    NormalPoint: prop<NormalPoint[]>(() => []),
}) implements IResultCloud {};

export default ResultCloud;

File: packages/csharp/onto/OnToSharp/src/Types/ResultCloud.cs
using GraphQLCodeGen;
using Stratosphere;


namespace Types {

    public class ResultCloud : OntoNode, Schema.ResultCloud {
        public ResultCloudData ResultCloudData { get; set; }
        public List<NormalPoint> NormalPoint { get; set; }

        public ResultCloud(ResultCloudData? resultCloudData, List<NormalPoint>? normalPoints) {
            this.NormalPoint = normalPoints ?? new List<NormalPoint>();
            this.ResultCloudData = resultCloudData ?? new ResultCloudData();
        }
        
    }

}
